PhysicsTools/NanoAOD/python/nanoTP_cff.py

```diff
@@ -76,3 +79,14 @@ def customizeNANOTP(process):
     process.finalMuons.cut = cms.string("||".join([passStandalone, process.finalMuons.cut.value()]))
     process.linkedMuons.cut = process.finalMuons.cut
     return process
+
+def nanoGenWmassCustomize(process):
```

Comment:
You don't actually need this here, you can reference the customise from nano_cff.py in cmsDriver